plugin.video.ad6/README.md

```diff
@@ -0,0 +1,13 @@
+# Welcome to your addon

```

Comment:
This could also be removed (it is just a sample readme)

Reply:
Done